project/plugins.sbt

```diff
@@ -15,3 +15,6 @@ libraryDependencies += { "org.scala-sbt" % "scripted-plugin" % sbtVersion.value
 
 // addSbtPlugin("de.heikoseeberger" % "sbt-header" % "1.5.1")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "1.5.1-2-g8b57b53")
+
+addSbtPlugin("com.lucidchart" % "sbt-scalafmt-coursier" % "1.15")
```

Comment:
should we be using `sbt-scalafmt` instead? i wasnt aware able to find coursier being used in this project.

Reply:
According to https://github.com/lucidsoftware/neo-sbt-scalafmt#usage `if you use coursier, you must use sbt-scalafmt-coursier`